includes/Integrations/Core_Themes_Support.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * Class Core_Themes_Support
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Integrations;
+
+use Google\Web_Stories\Customizer;
+
+/**
+ * Class Core_Themes_Support.
+ */
+class Core_Themes_Support {
+
+	/**
+	 * Current theme
+	 *
+	 * @var string
+	 */
+	protected $template = '';
+
+	/**
+	 * Default array of core themes to add support to.
+	 *
+	 * @var array
+	 */
+	protected static $supported_themes = [
+		'twentytwentyone',
+		'twentytwenty',
+		'twentynineteen',
+		'twentyseventeen',
+		'twentysixteen',
+		'twentyfifteen',
+		'twentyfourteen',
+		'twentythirteen',
+		'twentytwelve',
+		'twentyeleven',
+		'twentyten',
+		'classic',
+		'default',
+	];
+
+	/**
+	 * Get list of supported core themes.
+	 *
+	 * @since 1.3.0
+	 *
+	 * @return string[] Slugs for supported themes.
+	 */
+	public static function get_supported_themes() {
+		return self::$supported_themes;
+	}
```

Comment:
Why does this need to be a static function?

Reply:
```suggestion
	public function get_supported_themes() {
	    $themes = [
		   'twentytwentyone',
		   'twentytwenty',
	       'twentynineteen',
		   'twentyseventeen',
		   'twentysixteen',
		   'twentyfifteen',
		   'twentyfourteen',
		   'twentythirteen',
		   'twentytwelve',
		   'twentyeleven',
		   'twentyten',
		   'classic',
		   'default',
	    ];
	    
		return $themes;
	}
```

I am not seeing the reason there is a class level property. Why not a function returning an array?

Reply:
Thoughts on adding a filter here? @swissspidy ?

Reply:
I am not sure adding a filter is necessary here, anyway we want to keep the support limited to core themes only. And if any other theme want to add support for the Web Stories customizer options then I think using `add_theme_support()` is much friendlier then adding filters.

Also I feel we can just change this to a static property on the class so that other devs or other integrations can directly access what core themes are added in support.

@swissspidy any thoughts?

Reply:
There maybe valid reason to add and remove themes from this list. For example, devs of the next core theme that is not on this list may want to add it to this list. You maybe using a child theme.

Reply:
I don't think this needs a filter. If anyone wants to add or remove a theme, they  can use `add_theme_support`/`remove_theme_support`.

But I also don't think this static property should be **public**, as it allows changing. I'd make  it private or at least protected.